general/management/commands/gm_worker_async_tasks.py

```diff
@@ -91,84 +78,93 @@ def task_whitelist_user(self, gearman_worker, gearman_job):
                     pending_ticket.status = TICKET_STATUS_CLOSED
                     pending_ticket.save()
 
-                message = "Whitelisted user: %s" % whitelist_user.username
-                self.write_stdout(message)
-                logger_async_tasks.info(message)
+                workers_logger.info("Whitelisted user (%s)" % json.dumps(
+                    {'user_id': whitelist_user.id,
+                     'username': whitelist_user.username,
+                     'work_time': time.time() - local_start_time}))
 
             count_done = count_done + 1
-            self.write_stdout("Finished processing one ticket, %d remaining" % (len(tickets)-count_done))
+
+        workers_logger.info("Finished whitelisting users from tickets (%s)" % json.dumps(
+            {'n_tickets': len(tickets), 'work_time': time.time() - start_time}))
```

Comment:
this will have lots of decimal places, normally when I do something like this I use `round()`

Reply:
Nice one! Addressed in c4ef4707a3719bb7ac444dbf2905eb13bea18109